peerserver.js

```diff
@@ -2,89 +2,73 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-// Prepare for web server
-var fs = require("fs");
-var path = require("path");
-var url = require('url');
-var config = require('./config');
-var account = require('./vendormodule');
+'use strict';
 
-var dirname = __dirname || path.dirname(fs.readlinkSync('/proc/self/exe'));
-var httpsOptions = {
+// Prepare for web server.
+const fs = require("fs");
+const path = require("path");
+const url = require('url');
+const config = require('./config');
+const forwardEventName = 'owt-message';
+
+const dirname = __dirname || path.dirname(fs.readlinkSync('/proc/self/exe'));
+const httpsOptions = {
```

Comment:
Do we need to add a hard requirement of forcing TLS1.2 here?

Reply:
It depends on UA's behavior. @HuaChunbo may have suggestion on this?